### ISSUE TITLE
Date Picker: Calendar value is not set with Reactive Form and emit false 

### ISSUE THREAD

**Issue by boeckMt** [vmware/clarity#4748]
## Describe the bug

When the Date Picker is used with Reactive Forms and the value of the Date Picker is set from the Form Control with emit false  `FormControl.setValue(value, { emitEvent: false })` then only the input field is updated but not the Calendar.

## How to reproduce

https://stackblitz.com/edit/date-picker-forms-clarity-v3-dark-theme?file=src%2Fapp%2Fapp.component.html

1.  Set the date with one Date Picker (then the second Date Picker is updated to the same value)
2.  Click on 'Toggle datepicker' on the second Date Picker and there the value is not updated in the Calendar.

## Expected behavior

If the value of the Input is set (when emit is false) also the Calendar should be set to the same value

## Versions

**App**

- Angular: [^9.0.4]
- Clarity: [3.0.0]

**Device:**
- Browser [chrome, firefox]

## Additional notes
Even if an approach of `[formControl]="dateControl1"  [value]="dateControl1.value"` is used like described in [syncing-multiple-reactive-form-inputs-in-angular](https://coryrylan.com/blog/syncing-multiple-reactive-form-inputs-in-angular) the value is not set.

If the Form Control has `emit: true` then the value of the Calendar is set, but a subscription to the formGroup is triggered twice. 

The case is to update/synchronize different date inputs to one value, e.g. Date Pickers and a Range Slider which should be in sync.

**Comment by hippee-lee** [comment]
Hi Mathias,

When I change the example over to `emit: true` I am not seeing the FormGroup emit 2x per date change. 

Here is a forked version of the example: https://stackblitz.com/edit/gh-4748-reactive-datepicker-syncs
Am I missing something?

**Comment by boeckMt** [comment]
Hello @hippee-lee,

first thanks for the fast response!

You're right, the FormGroup is emitting only once because i filtered the events with `distinctUntilKeyChanged`.
This was my workaround to prevent the Form from emitting duplicates, but if i set `emit: false`, the Form only emits once without this filtering.

So I was only surprised why the value is set on the input but not in the Calendar and it is kind of a low priority Issue.

Best regards
Mathias

**Comment by hippee-lee** [comment]
Hmm, sorry I'm not an expert with forms.
It seems like when date 1 is set the form would emit that change and then when the observer on date 1 sets the date 2 value the form would emit that change. But It looks like both things emitted are the same. 

I'll mark this to look into.
